Makefile

```diff
@@ -107,7 +107,13 @@ pkg-build-wheel:
 	poetry build --format wheel
 	for pkg in $(PKG_SET); do cd $$pkg && poetry build --format wheel && cd ../..; done
 
-	poetry freeze-wheel
+	poetry freeze-wheel || { \
```

Comment:
so this can error for other reasons then non existence ie no locks, lock not up to date, etc. I think a more specific check to is it installed is probably more appropriate. ie `poetry self show plugins | grep freeze`